docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md

```diff
@@ -405,3 +405,142 @@ Algorithm       Hash
 ---------       ----                                                              ----
 SHA256          F46597C0C63A1EEFB200D40EDF654E52F10C3D5D21565886AD603FABAF8D39FB  E:\test_file.txt
 ```
+
+## USB004.001 Enable USB stack (Firmware)
```

Comment:
I think these test cases should be localized at the top of the test suite (this is the basic functionality).

Reply:
Moved